libgssapi/src/credential.rs

```diff
@@ -184,6 +187,7 @@ impl Cred {
         }
     }
 
+    #[cfg(feature = "s4u")]
     pub fn store(
```

Comment:
The things used in this function didn't exist on my Mac build of libgssapi-sys, so I assumed they were maybe supposed to be behind the same feature that's used in some othe r parts of this file.